tests/src/test/java/alluxio/client/rest/S3ClientRestApiTest.java

```diff
@@ -953,7 +955,11 @@ private void putObjectTest(String bucket, String objectKey, byte[] object, Strin
   }
 
   @Test
+  @Ignore
   public void testGetDeletedObject() throws Exception {
+    // This test requires the following property key change
+    // Configuration.set(PropertyKey.PROXY_S3_WRITE_TYPE, WriteType.CACHE_THROUGH.name());
+
```

Comment:
@Xenorith I have to skip this unit test because it depends on UFS writes; which if enabled result in issues with the Authorization-related tests because this class uses a local UFS.